Softeq.XToolkit.WhiteLabel.Droid/MainApplicationBase.cs

```diff
@@ -76,10 +76,18 @@ protected virtual void StartScopeForIoc()
         {
             var containerBuilder = new ContainerBuilder();
             ConfigureIoc(containerBuilder);
+            RegisterServiceLocator(containerBuilder);
             RegisterInternalServices(containerBuilder);
             Dependencies.IocContainer.StartScope(containerBuilder);
         }
 
+        protected virtual void RegisterServiceLocator(ContainerBuilder builder)
+        {
+            var serviceLocator = new IocContainer();
+            Dependencies.Initialize(serviceLocator);
```

Comment:
Please fix `RegisterServiceLocator` method to prevent exception in the existed projects. If you override method and call base you will have an exception in `Dependencies.Initialize` method that `ServiceLocator` was already registered. Please do the following code only if `servicelocator` not registered

Reply:
Updated